src/geometric_utils.ts

```diff
@@ -1,5 +1,5 @@
-const turf = require("@turf/turf");
-const polygonClipping = require("polygon-clipping");
+import * as turf from "@turf/turf";
+import * as polygonClipping from "polygon-clipping";
```

Comment:
ESLint wanted these as imports, after which it made a large number of complaints about type mismatches.

I operated under the assumption that the types had already been sorted out, and casted everything accordingly.

I also unwrapped a lot of function definitions because they exceed 120 characters, which is longer than even I could justify. Owners of ultrawide monitors may still protest.